docs/api/remote-procedure-calls-raw-transactions.md

```diff
@@ -1144,6 +1144,68 @@ Result:
 
 _See also_
 
+* [GetRawTransactionMulti](../api/remote-procedure-calls-raw-transactions.md#getrawtransactionmulti): gets hex-encoded serialized transactions or a JSON object describing the transactions.
+* [GetSpecialTxes](../api/remote-procedure-calls-blockchain.md#getspecialtxes): returns an array of special transactions found in the specified block
+* [GetTransaction](../api/remote-procedure-calls-wallet.md#gettransaction): gets detailed information about an in-wallet transaction.
+
+## GetRawTransactionMulti
+
+The [`getrawtransactionmulti` RPC](../api/remote-procedure-calls-raw-transactions.md#getrawtransactionmulti) gets hex-encoded serialized transactions or a JSON object describing the transactions. This RPC returns the same type of information as the [`getrawtransaction` RPC](../api/remote-procedure-calls-raw-transactions.md#getrawtransactionmulti).
+
+_Parameter #1---block hashes and transaction hash list_
+
+| Name | Type         | Presence                | Description |
+| ---- | ------------ | ----------------------- | ----------- |
+| Transactions | object | Required<br>(exactly 1) | A JSON object with block hashes as keys and lists of transaction hashes as values (no more than 100 in total) |
+|→<br>Block Hash | array | Required<br>(1 or more) | The block hash and the list of transaction ids to fetch |
```

Comment:
If there's provided `0` as a block-hash, the transaction from mempool will be returned.

Reply:
Does https://github.com/dashpay/docs-core/pull/97/commits/bad755b7e42360ea5e704083c8e8d1824be5ada5 look better?